release-notes/General/General_Feature_Release_10.3/General_Feature_Release_10.3.10.md

```diff
@@ -30,6 +31,20 @@ Apart from **ID only** scripts, which use the `OnDomInstanceCrud` entry point me
 
 For more detailed information, see [ExecuteScriptOnDomInstanceActionSettings](xref:ExecuteScriptOnDomInstanceActionSettings).
 
+#### Support for real-time GQI row updates [ID_37060]
+
+<!-- MR 10.4.0 - FR 10.3.10 -->
+
+Real-time row updates are now supported for GQI session results for specific data sources and operators. This means that, when this is supported in the client, real-time updates can be displayed for row additions, modification, or deletions.
+
+At present, this is supported for the following GQI data sources:
+
+- Parameter table (except partial tables)
+- Views
+- Ad-hoc data
```

Comment:
I'd leave out the ad-hoc data source for now as we want to improve this further before releasing it to the public.